packages/shared/src/contexts/AlertContext.tsx

```diff
@@ -93,20 +97,39 @@ export const AlertContextProvider = ({
     },
   );
 
+  const { mutateAsync: updateLastBootPopup } = useMutation(
+    () => request(graphqlUrl, UPDATE_LAST_BOOT_POPUP),
+    {
+      onMutate: () =>
+        updateAlerts({
+          ...alerts,
+          lastBootPopup: new Date(),
```

Comment:
Do we still need this new Date() here?

Reply:
Its the optimistic update, lets leave it for now.
Oh we should actually add the state itself then this doesnt matter even 👍

Reply:
It is just optimistic update I think.